### PR TITLE
core-1687 Add resource usage summary and plan information to dashboard.

### PR DESCRIPTION
**Pull request by sri2k1us**
image

image

image

### CODE REVIEW

File: src/components/dashboard/constants.js [hunk 1]
```diff
@@ -46,4 +46,7 @@ export const XS_NUM_COLUMNS = 1;
 export const XS_PIXELS = 0;
 
 export const USER_SURVEY = "https://cyver.se/srvy";
-export const PHX_BIO_URL = "https://www.phoenixbioinformatics.org/";
\ No newline at end of file
+export const PHX_BIO_URL = "https://www.phoenixbioinformatics.org/";
+export const FEATURE_MATRIX_URL = "https://cyverse.org/subscribe";
+export const CHECKOUT_URL =
+    "https://cyverse-subscription-sandbox.phoenixbioinformatics.org";
```
Comment (sri2k1us) [inline src/components/dashboard/constants.js:52]: This is sandbox for testing Phx Bio Credit card payment site.

File: src/components/dashboard/constants.js [hunk 2]
```diff
@@ -46,4 +46,7 @@ export const XS_NUM_COLUMNS = 1;
 export const XS_PIXELS = 0;
 
 export const USER_SURVEY = "https://cyver.se/srvy";
-export const PHX_BIO_URL = "https://www.phoenixbioinformatics.org/";
\ No newline at end of file
+export const PHX_BIO_URL = "https://www.phoenixbioinformatics.org/";
+export const FEATURE_MATRIX_URL = "https://cyverse.org/subscribe";
```
Comment (sri2k1us) [inline src/components/dashboard/constants.js:50]: This page doesn't exist yet but it will be available before this change goes to prod.

**Review by johnworth — APPROVED** [review]
LGTM

**Review by psarando — COMMENTED** [review]
LGTM 👍 
I would refactor the Checkout URL into a config, though.

File: src/components/dashboard/constants.js (hunk 1 above)
Comment (sri2k1us) [reply src/components/dashboard/constants.js:52]: Good point. But as far as I know, the `Buy` button here is only for QA env to test. For prod, the `Buy` button will be in main website subscription page  (https://cyverse.org/subscribe)(currently being built by Mariah). So I will delete the `Buy` button once that page is live.

**Pull request merged**